portfolio/src/main/webapp/style.css

```diff
@@ -123,37 +159,113 @@ main#landing > #hero{
         opacity: 0;
     }
     to {
-        bottom: -65px;
+        bottom: -20px;
         opacity: 1;
     }
 }
 
+main#projects {
+    display: flex;
+    flex-direction: column;
+}
 
+main#projects > #projects-list {
+    width: 85%;
+    height: 0;
+    flex-grow: 1;
 
-main#landing > #watermark{
-    position: absolute;
-    left: calc(100% - 60px);
-    bottom: 60px;
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    align-content: flex-start;
+}
 
-    transform: rotate(-90deg);
-    transform-origin: bottom left;
+main#projects > #projects-list > .project{
+    flex-basis: 250px;
+    margin-bottom: 64px;
+}
 
+main#projects > #projects-list > .project > .name {
     font-family: Montserrat;
-    font-size: 12.5vh;
-    line-height: 10vh;
-    color: #e6e6e6;
+    font-size: 24px;
     font-weight: 600;
+    color: black;
+}
 
-    animation: watermark-entry 1.4s forwards ease-out;
+main#projects > #projects-list > .project > .description {
+    font-family: Montserrat;
+    font-size: 16px;
+    font-weight: 500;
+    color: #787878;
+
+    margin-top: 8px;
+    width: 80%;
 }
 
-@keyframes watermark-entry{
-    from {
-        margin-bottom: -60px;
-        opacity: 0;
-    }
-    to {
-        margin-bottom: 0px;
-        opacity: 1;
-    }
+main#projects > #projects-list > .project > .tags {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+
+    margin-top: 16px;
+}
+
+main#projects > #projects-list > .project > .tags > .tag {
```

Comment:
This is fine, but I do appreciate your desire for something that reads a bit cleaner.

Reply:
I imagine those selectors will get old pretty fast :) Will migrate to SASS!